hazelcast/test/src/HazelcastTests8.cpp

```diff
@@ -2108,6 +2108,20 @@ namespace hazelcast {
     }
 }
 
+namespace hazelcast {
+    namespace client {
+        namespace test {
+            class ClientMessageTest: public ClientTestSupport {};
```

Comment:
PR looks good. Can you also add a test with an actual codec, e.g. `MapSizeCodec`? Encode a request and call getOperationName method.